lib/install/jest.rb

```diff
@@ -0,0 +1,39 @@
+require "webpacker/configuration"
+
+say "Copying the example test files"
+copy_file "#{__dir__}/examples/jest/hello_jest.js",
+          "#{Webpacker.config.source_entry_path}/hello_jest.js"
+copy_file "#{__dir__}/examples/jest/hello_jest.spec.js",
+          "#{Webpacker.config.root_path.join(tests_folder)}/hello_jest.spec.js"
```

Comment:
It looks like `test_folder` is not defined yet, that happens in line 13, or am I missing something?

Reply:
you are not. I updated the jest.rb. I guess a set of tests wouldn't hurt here, right? It would have been nice to have some examples but there are not. If you reckon, I can write the first one here.